aries-site/src/data/wcag/tip.json

```diff
@@ -0,0 +1,512 @@
+    "notes": ""
+  },
+  {
+    "rule": "2.3.1",
+    "label": "Three Flashes or Below Threshold",
+    "Column1": "conditional",
+    "notes": "Dependent on tip animation"
+  },
+  {
+    "rule": "2.3.2",
+    "label": "Three Flashes",
+    "Column1": "conditional",
+    "notes": "Dependent on tip animation"
+  },
+  {
+    "rule": "2.3.3",
+    "label": "Animation from Interactions",
+    "Column1": "conditional",
+    "notes": "Dependent on tip animation"
+  },
+  {
+    "rule": "2.4.1",
+    "label": "Bypass Blocks",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.2",
+    "label": "Page Titled",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.3",
+    "label": "Focus Order",
+    "Column1": "not-applicable",
+    "notes": "tip doesn't receive focus"
+  },
+  {
+    "rule": "2.4.4",
+    "label": "Link Purpose (In Context)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.5",
+    "label": "Multiple Ways",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.6",
+    "label": "Headings and Labels",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.7",
+    "label": "Focus Visible",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.8",
+    "label": "Location",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.9",
+    "label": "Link Purpose (Link Only)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.10",
+    "label": "Section Headings",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.2",
+    "label": "Pointer Cancellation",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.3",
+    "label": "Label in Name",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.4",
+    "label": "Motion Actuation",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.5",
+    "label": "Target Size (Enhanced)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.6",
+    "label": "Concurrent Input Mechanisms",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.7",
+    "label": "Dragging Movements",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.8",
+    "label": "Target Size (Minimum)",
+    "Column1": "not-applicable",
```

Comment:
Conditional based on the trigger used for tip

Reply:
@jcfilben Wouldn't this rule apply to the trigger only, not necessarily the tip itself? It seems like it applies indirectly to the tip. I don't have a hard opinion on this either way but just want to double check.

Reply:
Yeah that's fair I think it does apply to trigger only, I'm good leaving this as not-applicable